packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -13,7 +13,7 @@ Style guide: components.table
 */
 
 /*
-Table with two headers and a caption
+Table with multi headers and a caption
```

Comment:
Should we show a basic non-responsive table under the React section as well?

Reply:
Personally I think it would make the page really long, and the basic usage is very similar to the existing example